### PR TITLE
LPS-162118 [EVP] SiteInitializer-evp

### PR DESCRIPTION
**Pull request by Andrefes565**

### CODE REVIEW

**Comment by Andrefes565** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Author"
+			},
+			"name": "creator",
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:88]: Is this name creator or author?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+		},
+		{
+			"DBType": "Integer",
+			"businessType": "Integer",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "type_"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:158]: Please, change it.

```suggestion
				"en_US": "Type"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+		}
+	],
+	"objectLayouts": [
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "",
+	"parameterRequired": false,
+	"pluralLabel": {
+		"en_US": "EVPKaleoForkInstances"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:176]: Please, change it.

```suggestion
		"en_US": "EVP Kaleo Fork Instances"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+		"permissions": {
+		},
+		"publish": {
+		},
+		"update": {
+		}
+	},
+	"active": false,
+	"label": {
+		"en_US": "EVPRequestOrganization"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:18]: Please, change it.

```suggestion
		"en_US": "EVP Request Organization"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Author"
+			},
+			"name": "creator",
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:52]: Is this name creator or author?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "description"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:68]: Please, change it.

```suggestion
				"en_US": "Description"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "Integer",
+			"businessType": "Integer",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "nonprofitType"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:126]: Please, change it.

```suggestion
				"en_US": "Nonprofit Type"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "organizationId"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:144]: Please, change it.

```suggestion
				"en_US": "Organization Id"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "Boolean",
+			"businessType": "Boolean",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "shared"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:162]: Please, change it.

```suggestion
				"en_US": "Shared"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "subsidiaryGroupId"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:198]: Please, change it.
```suggestion
				"en_US": "Subsidiary Group Id"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "taxIdentificationNumber"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:216]: Please, change it.
```suggestion
				"en_US": "Tax Identification Number"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 12]
```diff
@@ -0,0 +1,266 @@
...
+		},
+		{
+			"DBType": "Boolean",
+			"businessType": "Boolean",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "verified"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:238]: Please, change it.

```suggestion
				"en_US": "Verified"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json [hunk 13]
```diff
@@ -0,0 +1,266 @@
...
+		}
+	],
+	"objectLayouts": [
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "",
+	"parameterRequired": false,
+	"pluralLabel": {
+		"en_US": "EVPRequestOrganizations"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:256]: Please, change it.

```suggestion
		"en_US": "EVP Request Organizations"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 14]
```diff
@@ -0,0 +1,250 @@
...
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Author"
+			},
+			"name": "creator",
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json:68]: It is created like this by default in Objects

[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json:68]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 15]
```diff
@@ -0,0 +1,250 @@
...
+	"objectFields": [
+		{
+			"DBType": "Double",
+			"businessType": "Decimal",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "amount"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 16]
```diff
@@ -0,0 +1,250 @@
...
+		},
+		{
+			"DBType": "Date",
+			"businessType": "Date",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "date_"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 17]
```diff
@@ -0,0 +1,250 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "evpGrantRequestId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 9 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 10 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 12 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 11 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 13 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 18]
```diff
@@ -0,0 +1,250 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "fileEntryId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json:142]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json [hunk 19]
```diff
@@ -0,0 +1,250 @@
...
+			"objectLayoutTabs": [
+			]
+		}
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "",
+	"parameterRequired": false,
+	"pluralLabel": {
+		"en_US": "EVPGrantTransactions"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpGrantTransaction.json:240]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 20]
```diff
@@ -0,0 +1,206 @@
...
+	"objectFields": [
+		{
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "abbreviation"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:30]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 21]
```diff
@@ -0,0 +1,206 @@
...
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Author"
+			},
+			"name": "creator",
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:72]: is the default of objescts

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 22]
```diff
@@ -0,0 +1,206 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "organizationId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:124]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 23]
```diff
@@ -0,0 +1,206 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "parentEVPDivisionId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:142]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 24]
```diff
@@ -0,0 +1,206 @@
...
+		}
+	],
+	"objectLayouts": [
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "control_panel.users",
+	"parameterRequired": false,
+	"pluralLabel": {
+		"en_US": "EVPDivisions"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:196]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json [hunk 25]
```diff
@@ -0,0 +1,206 @@
...
+		},
+		{
+			"DBType": "Integer",
+			"businessType": "Integer",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "type_"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpDivision.json:178]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 26]
```diff
@@ -0,0 +1,244 @@
...
+		"permissions": {
+		},
+		"publish": {
+		},
+		"update": {
+		}
+	},
+	"active": false,
+	"label": {
+		"en_US": "EVPEmployee"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:18]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 27]
```diff
@@ -0,0 +1,244 @@
...
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": false,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Author"
+			},
+			"name": "creator",
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:52]: is the default of objects

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 28]
```diff
@@ -0,0 +1,244 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "employeeUserId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:68]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 29]
```diff
@@ -0,0 +1,244 @@
...
+		},
+		{
+			"DBType": "Integer",
+			"businessType": "Integer",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "employmentType"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:86]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 30]
```diff
@@ -0,0 +1,244 @@
...
+		},
+		{
+			"DBType": "Date",
+			"businessType": "Date",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "hireDate"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:104]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 31]
```diff
@@ -0,0 +1,244 @@
...
+		},
+		{
+			"DBType": "String",
+			"businessType": "Text",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "ultiproEmployeeId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:212]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 32]
```diff
@@ -0,0 +1,244 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "subsidiaryEVPDivisionId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:194]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json [hunk 33]
```diff
@@ -0,0 +1,244 @@
...
+		}
+	],
+	"objectLayouts": [
+	],
+	"objectViews": [
+	],
+	"panelCategoryKey": "",
+	"parameterRequired": false,
+	"pluralLabel": {
+		"en_US": "EVPEmployees"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpEmployee.json:234]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 34]
```diff
@@ -0,0 +1,186 @@
...
+		"permissions": {
+		},
+		"publish": {
+		},
+		"update": {
+		}
+	},
+	"active": false,
+	"label": {
+		"en_US": "EVPKaleoForkInstance"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:18]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 35]
```diff
@@ -0,0 +1,186 @@
...
+	"objectFields": [
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "classNameId"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:32]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json [hunk 36]
```diff
@@ -0,0 +1,186 @@
...
+		},
+		{
+			"DBType": "Long",
+			"businessType": "LongInteger",
+			"defaultValue": "",
+			"indexed": true,
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "classPK"
```
[the remark this answers is not in the document]
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:50]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 6 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:68]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 7 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:126]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 8 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:144]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json (hunk 3 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpKaleoForkInstance.json:176]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json (hunk 5 above)
Comment (Andrefes565) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/evpRequestOrganization.json:52]: is the default of objects

**Review by silasbsi — APPROVED** [review]
LGTM :martial_arts_uniform:

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**